pettingzoo/mpe/group_spread_1p_1p_1p_v0.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, seed=None, local_ratio=0.5, max_frames=50, **env_args):
             0.0 <= local_ratio <= 1.0
         ), "local_ratio is a proportion. Must be between 0 and 1."
         scenario = Scenario()
-        world = scenario.make_world(groups=[1,1,1], num_colors=None, reward_per_group=True, randomise_all_colors=False, shuffle_obs_per_agent=True)
+        world = scenario.make_world(groups=[1, 1, 1], **env_args)
```

Comment:
Only set group argument in here (other consistent defaults are once defined in `group_spread.py`), and allow `env_args` to be given through to environment.

Reply:
Much cleaner and it will show in the config :+1: